.github/workflows/test.yml

```
@@ -5,14 +5,14 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node: [8, 10, 12, 14, 16]
+        node: [16, 18, 20]  # https://github.com/nodejs/release#readme
```

```suggestion
        node: [8, 16, 18, 20]
```

The link is unnecessary, and let's keep Node.js 8 IMO because dropping would be a semver-major change, which is not worth the noise for dependents until supporting Node.js 8 becomes a nuisance for us.

---

As someone who does [a lot of support of `node-gyp`](https://github.com/search?q=repo%3Anodejs%2Fnode-gyp+cclauss&type=issues), IMO it creates a lot of noise when dependents continue to require support for end-of-life software.  It also creates unjustifiable security risks.